src/integrationTest/java/uk/gov/hmcts/darts/cases/service/CloseOldCasesProcessorTest.java

```diff
@@ -245,6 +245,42 @@ void givenEventsUseLatestDateAsClosedDate() {
         assertEquals(RetentionPolicyEnum.DEFAULT.getPolicyKey(), caseRetentionEntity.getRetentionPolicyType().getFixedPolicyKey());
     }
 
+    @Test
+    void givenEventsUseLatestCurrentEventDateAsClosedDate() {
```

Comment:
The agreed test naming pattern for new tests is `methodName_expectation_conditions` e.g.  `getProductionOutputFiles_shouldSucceedAndReturnASingleItem_whenASuccessResponseIsReturnedFromArmWithASingularProductionExportFile`. 
Please could you update any tests created or edited to use this pattern?